Ali_Nurisiahsiahi.md

```diff
@@ -0,0 +1,7 @@
+I am Ali Nurisiahsiahi
+I like cycling,bodybuilding and walking because help my helth.
+My fouriy food are Spageti and Ghorma zabzi(Iranian food) becuse are delicious.
+My favorit song is Afrid and singer is Eminem https://www.youtube.com/watch?v=j5-yKhDd64s
+I am honest person and frindly.
+
+
```

Comment:
Empty lines at the end of the file always look suspicious for a code reviewer :) 

Read through your changes before you push a PR - it's like doing a small code review for your own code. It's a good practice that helps to spot and fix mistakes, typos, formatting issues, etc.

Reply:
Thank you and I need  someone knowing how to work with github.com.